python-avd/pyavd/_eos_designs/structured_config/connected_endpoints/utils.py

```diff
@@ -213,7 +213,7 @@ def _get_adapter_ptp(self: AvdStructuredConfigConnectedEndpoints, adapter: dict)
 
         ptp_config["enable"] = True
 
-        if get(adapter, "ptp.endpoint_role") != "bmca":
+        if get(adapter, "ptp.endpoint_role") != "dynamic":
```

Comment:
bmca is still supported - just deprecated.

```suggestion
        if get(adapter, "ptp.endpoint_role") not in ["dynamic", "bmca"]:
```